namespaces/Representor.md

```diff
@@ -0,0 +1,141 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Object Element)
+
+The Resource represents a Hypermedia Resource, with it's available relations, transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `content`: (object)
```

Comment:
Note, in MSON the `:` is used to denote a sample value e.g. `content: hello!`, it probably shouldn't be here